conf/_config.yml

```diff
@@ -81,3 +81,5 @@ kramdown:
 # exclude these directories from the build
 exclude:
     - static/css-src
+
+algolia_api_key: "0a916ab198bd93d031aa70611271e42e"
```

Comment:
General comment: should this be kept in the private SVN repo? If so, should we store a dummy value here to be replaced by committers building the site?